docs/app/pages/components/components-sections/panels/accordion/accordion.component.html

```diff
@@ -0,0 +1,37 @@
+<accordion [closeOthers]="true">
+
+    <accordion-group *ngFor="let group of groups" [(isOpen)]="group.open">
+
+        <ng-container accordion-heading>
+            <i class="hpe-icon pull-right" [class.hpe-chevron-left]="!group.open" [class.hpe-chevron-down]="group.open"></i>
+            <h4 class="m-nil">{{ group.heading }}</h4>
+        </ng-container>
+
+        <span>{{ group.content }}</span>
+
+    </accordion-group>
+
+</accordion>
+
+<hr>
+
+<p>
+    To create collapsible panels use the <code>accordion</code> tag. This contains the <code>closeOthers</code> property, 
```

Comment:
Could you use the same format as the other bootstrap sections?

"The Bootstrap `accordion` components can be used to create collapsible panels..."

Also I think we agreed not to document 3rd party components unless there is something that we have added. So perhaps the second paragraph can be replaced with a link to the 3rd party documentation instead. Main thing is the example and code snippet